### DISCUSSION TITLE
Unknown database error

### DISCUSSION THREAD

**Discussion by max-tet** [docker-library/ghost#368] [General]
The most simple docker install throws this error.

`docker run --rm -it ghost:alpine`

```
connect ECONNREFUSED 127.0.0.1:3306

"Unknown database error"
```

Then, the container exits with code 2.

**Comment by yosifkit** [comment]
Since `5.x`, ghost upstream has moved away from sqlite as a supported configuration, so an external database is required.

There is an example docker-compose file on the Docker Hub readme:
https://github.com/docker-library/docs/tree/94f25a1ea232fabf717d8871e9d30e691dd06c3a/ghost#-via-docker-stack-deploy-or-docker-compose

**Reply by max-tet** [reply]
Ok thanks! The dangers of using the `latest` tag. :wink: 

Perhaps it might be worthwhile to change the log to something more meaningful. "Database not configured, please provide configuration for external database" or something like that.
